### ISSUE TITLE
@hon2a/cypress-antd/lib/register missing

### ISSUE THREAD

**Issue by peterjurco** [hon2a/cypress-antd#4]
After updating the package to 0.4.0 I can no longer use the helpers as cypress commands as stated in the README:
```typescript
import '@hon2a/cypress-antd/lib/register'

// note that "child commands" are registered properly to automatically consume the yielded subject
cy.getButton('Uncool Button').shouldHaveTooltip('This button is so very sad.')
```

It's because there is no more `register.js` file that was previously responsible for adding the helpers as cypress commands. Will this be supported some time again or was it definitely scraped? If latter is true, it should be removed from the README. 

Btw. thanks @hon2a for your work, it's quite helpful 👏

**Comment by hon2a** [comment]
@peterjurco Thanks for the report. That certainly wasn't my intent, I'm just not using direct registration myself and clearly forgot to add a test case for it. Though that wouldn't solve a publish issue anyway. I'll look into it.

**Comment by hon2a** [comment]
Fixed in `0.4.10`.
